# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.68; am-1.11)
aclocal; autoconf; automake -a -c
./configure --prefix=

# install
install -d $DESTDIR/share/aclocal
install -m 644 xorg-macros.m4 $DESTDIR/share/aclocal
install -d $DESTDIR/lib/pkgconfig
install -m 644 xorg-macros.pc $DESTDIR/lib/pkgconfig

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/util-macros-build.pc << EOF
Name: util-macros
Version: 1.15.0
Description: X.Org m4 macros
Requires: $REQUIRES

devel=\\
/share/aclocal/xorg-macros.m4 \\
/lib/pkgconfig/xorg-macros.pc
EOF

exit
--- util-macros-1.15.0.orig/Makefile.am
+++ util-macros-1.15.0/Makefile.am
@@ -29,7 +29,7 @@
 install-data-hook:
 	rm -f $(DESTDIR)$(aclocaldir)/xorgversion.m4
 
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = xorg-macros.pc
 
 .PHONY: ChangeLog
